### PR TITLE
Gender-Display-Fix

### PR DESCRIPTION
**Pull request by karthik-sista**
### Jira Story

- [Display Age and Gender on the Create Relationship Form](https://osi-cwds.atlassian.net/browse/RL-189)

## Description
This PR is a bug fix for the above story. PGSQL has mixed data for genders, i.e M, male. It's not consistent, so using both the Gender enums to display genders on edit and create relationships cards 

## Tests
- [x] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/common/relationship/EditRelationshipForm.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import CheckboxField from 'common/CheckboxField'
 import DateField from 'common/DateField'
 import {RELATIONSHIP_TYPES} from 'enums/RelationshipTypes'
 import {GENDERS_LEGACY} from 'enums/Genders'
+import GENDERS from '../../enums/Genders'
```
Comment (NagaPradeepGajula-osi) [inline app/javascript/common/relationship/EditRelationshipForm.jsx]: lets keep the consistency we don't need the trials here.

File: app/javascript/common/relationship/EditRelationshipForm.jsx [hunk 2]
```diff
@@ -39,7 +40,7 @@ const EditRelationshipForm = ({onChange, person, relationship}) => {
               <ul className='unstyled-list'>
                 <li>{person.name}</li>
                 <li>{person.age}</li>
-                <li>{GENDERS_LEGACY[person.gender] || ''}</li>
+                <li>{GENDERS_LEGACY[person.gender] || GENDERS[person.gender] || ''}</li>
```
Comment (NagaPradeepGajula-osi) [inline app/javascript/common/relationship/EditRelationshipForm.jsx]: units test expectation for using these enum's though they are identical.

File: app/javascript/common/relationship/EditRelationshipForm.jsx (hunk 1 above)
Comment (karthik-sista) [reply app/javascript/common/relationship/EditRelationshipForm.jsx]: 👍  ya.

File: app/javascript/common/relationship/EditRelationshipForm.jsx (hunk 2 above)
Comment (karthik-sista) [reply app/javascript/common/relationship/EditRelationshipForm.jsx]: Added tests 👍

**Comment by karthik-sista** [comment]
retest this please.

**Comment by wmitchell-cambria** [comment]
> PGSQL has mixed data for genders

Is this only the case in Preint, or is it expected that this will occur in higher environments as well? Ideally, any one API endpoint would only return genders following a single format, and we would not need to implement this in the front-end.

Was there any investigation into solving the data problem rather than adding additional logic?

File: spec/javascripts/components/common/relationship/EditRelationshipFormSpec.jsx [hunk 3]
```diff
@@ -9,13 +9,13 @@ describe('EditRelationshipForm', () => {
     person: {
       name: 'Luke Skywalker',
       age: '20 yrs',
-      gender: 'M',
+      gender: 'M' || 'male',
```
[the remark this answers is not in the document]
Comment (karthik-sista) [reply spec/javascripts/components/common/relationship/EditRelationshipFormSpec.jsx]: yes. noticed it few mins ago. fixing it . thanks .

**Comment by karthik-sista** [comment]
Adding a Technical Debt story to fix the Bad data clean up in PGSQL . Merging this PR for temporary fix.
https://osi-cwds.atlassian.net/browse/RL-196

**Pull request merged**